src/db/models/json/attachment.ts

```diff
@@ -0,0 +1,80 @@
+import * as t from 'io-ts';
+import {
+  CASELOAD_VALUE as CASELOAD_ATTACHMENT_VALUE,
+  INDICATOR_VALUE as INDICATOR_ATTACHMENT_VALUE,
+} from './indicatorsAndCaseloads';
+
+const COST_ATTACHMENT_VALUE = t.intersection([
+  t.exact(
+    t.type({
+      cost: t.number,
+    })
+  ),
+  t.exact(
+    t.partial({
+      /**
+       * When neccesary, a cost breakdown can be provided with respect to a
+       * particular collection of objects.
+       *
+       * For example, when the object of this attachment is a governing entity,
+       * a breakdown needs to be provided for each of the global clusters,
+       * (even if the governing entity has 0 global clusters)
+       * and the total sum of the breakdown must match the overall cost when
+       * non-empty.
+       */
+      breakdown: t.array(
+        t.exact(
+          t.type({
+            objectId: t.number,
+            cost: t.number,
+          })
+        )
+      ),
+      // TODO: delete these properties once we've confirmed that they're not
+      // needed for any code that reads cost attachments
+      // (they seem to be produced by RPM frontend code related to other types)
+      /** @deprecated TODO: remove (this doesn't look like it's needed for cost attachments) */
+      name: t.string,
+      /** @deprecated TODO: remove (this doesn't look like it's needed for cost attachments) */
+      description: t.string,
+      /** @deprecated TODO: remove (this doesn't look like it's needed for cost attachments) */
+      metrics: t.exact(
+        t.partial({
+          values: t.exact(t.partial({})),
+          measureFields: t.array(t.UnknownRecord),
+        })
+      ),
+    })
+  ),
+]);
+
+export const ATTACHMENT_VERSION_VALUE = {
+  caseLoad: CASELOAD_ATTACHMENT_VALUE,
+  /**
+   * TODO
```

Comment:
Please write something in this TODO

`contact` seems to have following structure
```js
contact: t.type({
  leadAgency: t.string,
  contactName: t.string,
  contactEmail: t.string
})
```
but other keys are also available, like `'metrics'` and `'description'` :smile: like you defined for `COST_ATTACHMENT_VALUE`

Reply:
I think i'd rather leave this as `t.unknown` until we have a need to actually read or write data to these types of attachments using the v4 model, at which point we can test it more thoroughly. FOr now I will still update the TODO to say `TODO: Replace this with type that represents expected values`